src/debug/debugSession.ts

```diff
@@ -15,6 +16,8 @@ import { shell } from "../shell";
 import { DockerfileParser } from "../docker/dockerfileParser";
 import * as dockerUtils from "../docker/dockerUtils";
 
+const debugWikiUrl = "https://github.com/Azure/vscode-kubernetes-tools/blob/master/debug-on-kubernetes.md";

```

Comment:
Would prefer not to use the term 'wiki' for this.  Maybe `debugCommandDocumentationUrl`?